docs/assist-us/best-practices.md

```diff
@@ -50,7 +50,7 @@ Radio calls should take the following format:
 **STATION NAME**: "Message"
 ```
 
-!!! example
```

Comment:
This one might be better as an `example` since its highlighting how to format a radio call, rather than providing a phraseology case